src/properties/peerlistwidget.cpp

```diff
@@ -170,9 +173,9 @@ void PeerListWidget::showPeerListMenu(const QPoint&)
   QAction *dlLimitAct = 0;
 #endif
   QAction *banAct = 0;
-  QAction *copyIPAct = 0;
+  QAction *copyPeersAct = 0;
   if (!selectedPeerIPs.isEmpty()) {
-    copyIPAct = menu.addAction(IconProvider::instance()->getIcon("edit-copy"), tr("Copy IP"));
+    copyPeersAct = menu.addAction(IconProvider::instance()->getIcon("edit-copy"), tr("Copy selected peers"));
```

Comment:
Capitalize each word. "Copy Selected Peers"
Although some other neutral string should be used here. What if the user has selected only one peer? It would be weird to read "peers". Maybe "Copy Selected"?